rules/notion_rules/notion_workspace_settings_enforce_saml_sso_config_updated.py

```diff
@@ -0,0 +1,52 @@
+from global_filter_notion import filter_include_event
+from panther_base_helpers import deep_get
+from panther_notion_helpers import notion_alert_context
+
+
+def rule(event):
+    if not filter_include_event(event):
+        return False
+    return (
+        deep_get(event, "type", default="<NO_EVENT_TYPE_FOUND>")
+        == "workspace.settings.enforce_saml_sso_config_updated"
+    )
+
+
+def title(event):
+    user = deep_get(event, "actor", "person", "email", default="<NO_USER_FOUND>")
+    workspace_id = deep_get(event, "workspace_id", default="<NO_WORKSPACE_ID_FOUND>")
+    state = deep_get(
+        event,
+        "workspace.settings.enforce_saml_sso_config_updated",
+        "state",
+        default="<NO_STATE_FOUND>",
+    )
+
+    if state == "enabled":
+        return (
+            f"Notion User [{user}] updated settings to enable SAML SSO config "
+            f"to {state} for workspace id {workspace_id}"
+        )
+
+    return (
+        f"Notion User [{user}] updated settings to disable SAML SSO config "
+        f"to {state} for workspace id {workspace_id}"
+    )
+
+
+def severity(event):
+    state = deep_get(
+        event,
+        "workspace.settings.enforce_saml_sso_config_updated",
+        "state",
+        default="<NO_STATE_FOUND>",
+    )
+
+    if state == "enabled":
+        return "INFO"
+
+    return "HIGH"
```

Comment:
if `state` has an unexpected value, better to be  on the conservative (`HIGH`) side